arcus_quickstart.md

```diff
@@ -0,0 +1,114 @@
+<!--
+author:   Arcus Education
+email:    arcus-support@chop.edu
+version:  1.0.0
+language: en
+narrator: US English Female
+title: Arcus Quickstart
+
+link:  https://cdn.jsdelivr.net/gh/arcus/Arcus_Labs_Orientation@main/assets/styles.css
+link:  https://cdn.jsdelivr.net/gh/arcus/education_modules@main/assets/styles.css
+script: https://kit.fontawesome.com/83b2343bd4.js
+script: https://cdn.jsdelivr.net/npm/mermaid@9.4.3/dist/mermaid.min.js
+-->
+
+# Arcus Labs: Quick Start
+
+Ready to start a research project with Arcus, or have you been given a computational lab and want to know what to do next? This quick start guide will jump start your Arcus journey! 
+
+If you have questions about Arcus Labs that aren't answered here, please see our [Arcus Lab User Guide]() for more detailed documentation. 
+
+## Start a research project with Arcus
+
+To work with data provided by Arcus, you'll need an Arcus Computational Lab (called an Arcus Lab for short).
+
+When you [request an Arcus Lab](https://support.arcus.chop.edu/servicedesk/customer/portal/6/create/307), we evaluate your proposed study to make sure it's a good fit for our service, then build the computational environment and provision the data for your team so you can start work. 
+This process takes time and coordination across multiple groups within Arcus. 
+Here's what you can do to make the process as smooth as possible: 
+
+- Check first: Is your proposed study a good fit for Arcus?
+- Save time: Hone your data request beforehand
+- Know the rules: Security, privacy, and the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-use).
+
+### Is your proposed study a good fit for Arcus?
+
+<div style = "background-color:white;">
+
+<script style="display: block" run-once="true" modify="false">
+mermaid.initialize({});
+
```

Comment:
A few notes on the diagram:

* Mention NTP status in the "chop login credentials" e.g. "everyone has (or everyone can get, through NTP status) chop login"
* Have a descriptive no for each no, if possible, like, "no, there are some critical collaborators that I can't get NTP status for", "no, my project is operational or quality improvement, or otherwise not research", "no, we need a special tool that isn't supported by Arcus", etc.
* Make the tools question broader, like "we're able to use the Arcus toolkit, or we're willing to hire someone who can,"
* The final "not a good fit" should be softened: "Arcus might not be the right fit, depending on your circumstances." or similar...
* In short, we're aiming for helpful but not discouraging.
Also, non-human research is going to be supported in some degree in the future so stay tuned, I have an ask in to Marianne about this.

Reply:
UPDATE.  Remove entirely the "human data" question, we will and can accomodate animal models or wet lab projects.

OR, replace it with a question (we went back and forth on how to phrase this in the managers' meeting) that is something along the lines of "Can you describe your data clearly enough so that we can gather it for you"

Reply:
Another update / feedback from managers' meeting: Maybe change the tools question to "Are you prepared to clean and analyze data using Arcus tools" bc the tools are related to cleaning and analyzing data, not, for example, for data collection.  

Managers also want NTP info added to the first question (even though you talk about NTP later, just to make sure it doesn't turn people off too quickly.